tox.ini

```diff
@@ -12,11 +12,32 @@ extras =
     development
     pandas
 deps = pip
+external_wheels =
+    py37-ci: dist/*.whl
+    py38-ci: dist/*.whl
+    py39-ci: dist/.whl
+    py310-ci: dist/.whl
```

Comment:
Since we switched to using tox4, we don't use `tox-external-wheels` anymore
```suggestion
```

Reply:
updated